spec/unit/cli_spec.rb

```diff
@@ -626,6 +642,13 @@
         with(an_instance_of(SystemDescriptionStore), anything, anything)
       run_command(["list"])
     end
+
+    it "triggers the list task with --html switch" do
+      expect_any_instance_of(ListTask).to receive(:list) do |_instance, store, description, options|
```

Comment:
Unused block argument - `store`. If it's necessary, use `_` or `_store` as an argument name to indicate that it won't be used.<br>Unused block argument - `description`. If it's necessary, use `_` or `_description` as an argument name to indicate that it won't be used.